sermant-integration-tests/tag-transmission-test/httpclientv5-demo/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>tag-transmission-test</artifactId>
+        <groupId>io.sermant.tagtransmission</groupId>
+        <version>1.0.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>httpclientv5-demo</artifactId>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <httpcore.version>5.3.4</httpcore.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+            <version>${httpclient5x.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.core5</groupId>
+            <artifactId>httpcore5</artifactId>
+            <version>${httpcore.version}</version>
+        </dependency>
+    </dependencies>
```

Comment:
In httpclient repository 5.4.x depends on httpcore5-5.3.4, but in the maven repository depends on version 5.1.5, which will cause ClassNotFoundExcpetion at runtime, so I manually adjusted the version of httpcore5, otherwise it will not run, I guess that other users of httpclient5.4,5.5 will have the same problem.
<img width="680" alt="截屏2025-07-03 18 41 55" src="https://github.com/user-attachments/assets/c201d928-722f-43ea-9647-3830eeaf0689" />

https://github.com/apache/httpcomponents-client/blob/rel/v5.4.4/pom.xml#L65